Making help messages about PRs plural

bradmwilliams opened the pull request.

bparees commented on pkg/slack/slack.go:
```diff
@@ -80,7 +80,7 @@ func NewBot(botToken, botSigningSecret string, graceperiod time.Duration, port i
+		parser.NewBotCommand("launch <image_or_version_or_prs> <options>", &parser.CommandDefinition{
 			Description: fmt.Sprintf("Launch an OpenShift cluster using a known image, version, or PR. You may omit both arguments. Use `nightly` for the latest OCP build, `ci` for the the latest CI build, provide a version directly from any listed on https://amd64.ocp.releases.ci.openshift.org, a stream name (4.1.0-0.ci, 4.1.0-0.nightly, etc), a major/minor `X.Y` to load the \"next stable\" version, from ...
 				strings.Join(CodeSlice(manager.SupportedPlatforms), ", "),
 				strings.Join(CodeSlice(manager.SupportedParameters), ", ")),
```
there's an example one line below this of referencing a PR.  If you changed the example to illustrate referencing multiple PRs, i think that would be helpful (would illustrate the comma separated syntax).

similarly anywhere else we have examples that reference a PR.  I think it's easier for people to generalize an example of multiple items down to how to do it for one item, than the inverse.

bradmwilliams replied:
Added example to all commands that take options.

bradmwilliams commented:
/hold
Working on a few more changes

bradmwilliams commented:
I've updated the help messaging and formatting a bit and added a link to the FAQ.  Hopefully, this will be a little easier to read and digest for folks.
![help](https://github.com/openshift/ci-chat-bot/assets/43015045/b3fc68e0-8d46-4859-aaf6-932b0574ba58)

bparees commented:
/lgtm
